terraform/modules/gost_api/gateway.tf

```diff
@@ -69,23 +69,32 @@ module "api_gateway" {
 
   default_stage_access_log_destination_arn = join("", aws_cloudwatch_log_group.default[*].arn)
   default_stage_access_log_format = jsonencode({
-    requestId      = "$context.requestId"
-    ip             = "$context.identity.sourceIp"
-    requestTime    = "$context.requestTime"
-    httpMethod     = "$context.httpMethod"
-    routeKey       = "$context.routeKey"
-    status         = "$context.status"
-    protocol       = "$context.protocol"
-    responseLength = "$context.responseLength"
-    path           = "$context.path"
+    apiId           = "$context.apiId"
+    stage           = "$context.stage"
+    requestId       = "$context.requestId"
+    xrayTraceId     = "$context.xrayTraceId"
+    principalId     = "$context.authorizer.principalId"
+    requestTime     = "$context.requestTimeEpoch"
+    requestTimeCLF  = "$context.requestTime"
+    httpMethod      = "$context.httpMethod"
+    path            = "$context.path"
+    resourcePath    = "$context.resourcePath"
+    routeKey        = "$context.routeKey"
+    protocol        = "$context.protocol"
+    status          = "$context.status"
+    responseLength  = "$context.responseLength"
+    responseLatency = "$context.responseLatency"
+    ip              = "$context.identity.sourceIp"
+    userAgent       = "$context.identity.userAgent"
+    caller          = "$context.identity.caller"
+    user            = "$context.identity.user"
     integration = {
       error         = "$context.integration.error"
       serviceStatus = "$context.integration.integrationStatus"
       latency       = "$context.integration.latency"
       requestId     = "$context.integration.requestId"
       status        = "$context.integration.status"
       errorMessage  = "$context.integrationErrorMessage"
-      latency       = "$context.integrationLatency"
```

Comment:
This is the line that was causing the `tflint` check failures, since `lantency` is also defined on [line 94 (previously line 84)](https://github.com/usdigitalresponse/usdr-gost/pull/3481/files#diff-aac5295bae76e49b720fd47277cb4d113f53e00e5f71ab9bc7ce5aaefbb8cdaeL84) above. According to API Gateway [docs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-logging-variables.html), the `$context.integration.latency` and `$context.integrationLatency` variables are equivalent.